### PR TITLE
 add audit and some CI on ubuntu

### PR DESCRIPTION
**Pull request by MarcoIeni**
Initial work to solve #396 

- audit.yml checks if our dependencies have known security issues
- ci.yml checks if the code is well formatted and if some tests pass on ubuntu

Of course this does not cover everything jenkins covered before, but I prefer to go step by step :)

### CODE REVIEW

**Review by dandavison — APPROVED** [review]
LGTM!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
```
Comment (dandavison) [inline .github/workflows/ci.yml:20]: (Just noting -- ultimately I think we want `make test` to run)

**Comment by dandavison** [comment]
Looks like that worked perfectly: https://github.com/dandavison/delta/actions/runs/377690244 (do you have access to that page?)

<table><tr><td>image</td></tr></table>

**Comment by MarcoIeni** [comment]
yes, I have access :)

**Pull request merged**